### ISSUE TITLE
Better method for updating scripts

### ISSUE THREAD

**Issue by pshriwise** [ChristinaLK/python-novice-gapminder-custom#5]
Got a lot of feedback about it being difficult to listen and update scripts in during the "Command Line Programs" through "Running Scripts and Importing" lessons. 

These all work with the same script. One proposed solution at the end of the workshop was to put the different versions of the script into a git repo and make branches for each version we will run during the lessons. 

IMO the best way to do this is to have them follow the editing process performed only by the instructor on-screen to see things happen in action. And then after the instructor commits, checkout the appropriate branch and run the code. 

Upside: They have to worry about editing the text/code less
Downside: They will interact with branches more than with the more common change/add/commit/ development cycle.

**Comment by ChristinaLK** [comment]
Maybe providing a color-coordinated version of the script?  So people can add it themselves from a file?  The branch idea sounds kind of clunky.  :(

**Comment by pshriwise** [comment]
I agree that a color-coordinated version of the script would be great, but I don't quite follow when you say add it themselves from a file... as in different files for each version of the script that will be run?

I think you're probably right about the branches though :( Clunky and might give the wrong idea about how branches should be used.

**Comment by ChristinaLK** [comment]
Sorry, that comment was unclear -- color coordinated so people could add from a *handout* or some other visual and didn't necessarily have to follow you on-screen to add the text.

**Comment by pshriwise** [comment]
I like it. Could provide that highlighting in the lesson site and use that same markdown to create handouts. 

This is probably going to mean some special editing of a jekyll plugin...

**Comment by pshriwise** [comment]
I like it. Could provide that highlighting in the lesson site and use that same markdown to create handouts. 

This is probably going to mean some special editing of a jekyll plugin...

**Comment by ChristinaLK** [comment]
Yeah, I don't know if you can really control colors/highlighting easily in markdown.  :((((
